docs/resource/aws_iam_role_policies_exclusive.md

```diff
@@ -0,0 +1,52 @@
+---
+subcategory: "IAM (Identity & Access Management)"
```

Comment:
I can ask @guineveresaenger if there's a better way to inject a note, I've not found quickly.

Reply:
Yeah! To shadow @flostadler's concern, doing it this way makes the doc instantly stale. Sometimes this is an OK choice; I don't know the context for this resource.

To inject a note, I recommend using [Edit Rules](https://github.com/pulumi/pulumi-terraform-bridge/blob/master/docs/guides/docs-edits.md#edit-rules). I'm happy to help but you can write a string or regex based find/replace that pins a note to the main text of the doc (the "Description" section in the schema).

With EditRules, you can also add a hard fail on schemagen in case your find/replace condition is a bit too fragile.

Reply:
Ok I'll take a look!

Reply:
This doesn't work because of a bridge bug: https://github.com/pulumi/pulumi-terraform-bridge/issues/2633